src/common/utils.ts

```diff
@@ -26,17 +46,37 @@ export function createForm(payload: string | Payload): FormData {
   const form = new FormData();
 
   if (typeof payload === 'string') {
-    form.append('image', createReadStream(payload));
+    form.append('image', payload);
     return form;
   }
 
   for (const [key, value] of Object.entries(payload)) {
-    if (key === 'image' || key === 'video') {
-      if (!payload.type || payload.type === 'file')
-        form.append(key, createReadStream(value));
+    const supportedUploadObjectTypes = ['base64', 'stream'];
+    if (supportedUploadObjectTypes.indexOf(key) !== -1) {
+      if (supportedUploadObjectTypes.indexOf(payload.type as string) !== -1) {
+        form.append(key, payload);
+      }
     } else {
       form.append(key, value);
     }
   }
   return form;
 }
+
+export function getImgurApiResponseFromResponse(
```

Comment:
is the idea that this replace `resolveBodyOnly` prop from `got`?

Reply:
@kaimallea I suppose so. I feel that a helper method here makes the package a bit more modular. This might be solvable by an option with axios that I'm not aware of.